### PR TITLE
src: teach llnode how to print Symbol properties names

### PR DESCRIPTION
**Pull request by mmarchini**
Still need to add the metadata to V8 before landing this.

Closes: https://github.com/nodejs/llnode/issues/156

### CODE REVIEW

File: src/llv8.cc [hunk 1]
```diff
@@ -955,6 +961,15 @@ std::string HeapNumber::Inspect(Error& err) {
 }
 
 
+std::string Symbol::ToString(Error& err) {
+  if (!String::IsString(v8(), Name(err), err)) {
```
Comment (cjihrig) [inline src/llv8.cc]: Any reason not to do `Symbol()`/`Symbol('foo')`, like would be shown in JS?

Comment (mmarchini) [reply src/llv8.cc]: You mean for unnamed symbols? `Symbol()` sounds like a good idea in this case.

**Comment by mmarchini** [comment]
Rebased, changed display format to `Symbol()/Symbol('foo')` and added code to fallback to `<non-string>` when Symbols metadata is not available (Node.js 6 and 8).

**Comment by mmarchini** [comment]
Landed in e30beba

**Pull request closed without merging**